batavia/modules/time.js

```diff
@@ -1,7 +1,9 @@
 
 batavia.modules.time = {
+    _startTime: new Date().getTime(),
     clock: function() {
-        return new Date().getTime();
+        // TODO: this does something a little different in CPython.
```

Comment:
Reading through the docs for `time.clock`, it is allowed to return either the CPU time or the real time elapsed. So, this is technically within the specs.

I'll delete the comment.